src/components/Dashboard/TodoList.vue

```diff
@@ -44,21 +39,11 @@
           <div class="list-item">
             <div class="todo-label">
               <!-- title -->
-              <h3 class="list-item__title">{{item.title}}
-                <v-btn v-if="isStatusDraft(item) && isTypeCorrection(item)"
-                  class="expand-btn ml-0"
-                  outlined
-                  color="red"
-                  :ripple=false
-                >
-                  <v-icon left>mdi-information-outline</v-icon>
-                  <span>{{ (panel === index) ? "Hide Details" : "View Details" }}</span>
-                </v-btn>
```

Comment:
This button should not have been in the H3 and is now moved to the "draft" section below.